buttercup.el

```diff
@@ -814,39 +895,13 @@ Do not change the global value.")
 ;;;;;;;;;;;;;
 ;;; Reporters
 
-(defvar buttercup-reporter #'buttercup-reporter-adaptive
```

Comment:
I'm somewhat unhappy with moving this variable out of the "Reporters" section. It belongs here. Sadly, the block can not be moved as a whole, because it itself needs previous code. Elisp should be smarter about "undefined variable" references, really. (Good example where stupid linter rules reduce clarity, instead of increasing it.)